docs/hub/_toctree.yml

```
@@ -125,12 +125,34 @@
     title: Dataset Cards
   - local: datasets-gated
     title: Gated Datasets
+  - local: datasets-adding
+    title: Adding a new Dataset
   - local: datasets-viewer
     title: Dataset Viewer
-  - local: datasets-usage
-    title: Using Datasets
-  - local: datasets-adding
-    title: Adding New Datasets
+    sections:
+      - local: datasets-viewer-configure
+        title: Configure the Dataset Viewer
+  - local: datasets-data-files-configuration
+    title: Data files Configuration
+    sections:
+      - local: datasets-file-names-and-splits
+        title: File names and splits
+    sections:
+      - local: datasets-manual-configuration
+        title: Manual Configuration
```

```suggestion
    sections:
      - local: datasets-file-names-and-splits
        title: File names and splits
      - local: datasets-manual-configuration
        title: Manual Configuration
```

doc preview is failing because `sections` was declared multiple times here